docs/source/conf.py

```diff
@@ -108,7 +108,15 @@ def ultimate_replace(app, docname, source):
     source[0] = result
 
 
-ultimate_replacements = {"|release|": release}
+SQLFLUFF_RELEASE_FLAG = "|release|"
+SQLFLUFF_CODE_SPACE_NUM = 4
+SQLFLUFF_SQL_CODE_FLAG = f"{' ' * SQLFLUFF_CODE_SPACE_NUM}.. code-block:: sql"
+SQLFLUFF_SQL_CODE_FORCE_FLAG = f"{SQLFLUFF_SQL_CODE_FLAG}\n{' ' * (SQLFLUFF_CODE_SPACE_NUM + 3)}:force:"
+
+ultimate_replacements = {
+    SQLFLUFF_RELEASE_FLAG: release,
+    SQLFLUFF_SQL_CODE_FLAG: SQLFLUFF_SQL_CODE_FORCE_FLAG
+}
```

Comment:
Cann't found any config for global `:force:` option, but I found out we could use function `ultimate_replace` to make it

Reply:
cc: @tunetheweb

Reply:
I can't get this to work.

I'm doing the following:
```
cd docs
pip install -r requirements.txt
make html
python -m http.server --directory build/html
```

But none of the ultimate replacements are replace and I see alerts in the `make html`.

Am I doing something wrong?

BTW a README for this docs directory would be very useful to tell people how to build and test these sorts of changes.

Reply:
Oh…… my bad, I run `python setup.py install` with commit https://github.com/sqlfluff/sqlfluff/pull/1393/commits/e94c179bc9923e28511229ec386aa6489b1a41c3, so that it would render with `:force:` option

Reply:
I'm confused. Then I should run the following?:

```
python setup.py install
cd docs
pip install -r requirements.txt
make html
python -m http.server --directory build/html
```

That still doesn't work for me?

Reply:
Yeah, It's should not work event in my laptop . It's work in my env by accident, cause my docstring with `:force:` option, and just commit https://github.com/sqlfluff/sqlfluff/commit/e94c179bc9923e28511229ec386aa6489b1a41c3 look like.

BTW, I test the doc by

```sh
pip install sqlfluff
python setup.py install
cd docs
pip install -r requirements.txt
make html
```

and then just double click the `index.html` file